tools/proofers/spellcheck_text.inc

```diff
@@ -374,3 +377,246 @@ function spellcheck_apply_corrections()
+    if (count($languages) > 0) {
+        echo sprintf(_("Dictionaries used: <b>%s</b>."), implode(', ', $languages));
+    } else {
+        echo "<b>" . _("No check against a dictionary has been made.") . "</b>";
+    }
+
+    // for the vertical layout, stick in a line break
+    // for the horizontal one, stick in a space
+    if ($user->profile->i_layout == 1) {
+        echo "<br>";
+    } else {
+        echo " ";
+    }
+
+    // output the code allowing the user to select another language
+    echo _("Use additional: ");
+    echo "<select name='aux_language'>";
+    echo "<option value=''>" . _("Language") . "</option>\n";
+
+    // get a list of languages with dictionaries installed on the system
+    $dict_list = get_languages_with_dictionaries();
+    asort($dict_list);
+    foreach ($dict_list as $langcode => $language) {
+        // skip the languages that we just used.
+        if (in_array($language, $languages)) {
+            continue;
+        }
+        echo "<option value='" .  attr_safe($language) . "'";
+        if (strcmp($language, $aux_language) == 0) {
+            echo " selected";
+        }
+        echo ">$language</option>\n";
+    }
+    echo "</select>";
+
+    $value = attr_safe(_("Check"));
+    $title = attr_safe($word_check_messages["rerun"]);
+    echo "<input type='submit' name='rerunauxlanguage' id='rerunauxlanguage' value='$value' title='$title'>";
+    echo "<br>";
+
+    // show help blurb on the UA&S icon
+    echo sprintf(
+        _("The %s icon accepts the word for this page and suggests it for the dictionary."),
+        "<img src=\"$code_url/graphics/Book-Plus-Small.gif\" border=\"0\">"
+    );
+    echo "</p>";
+
+    // text highlight legend
+    if ($highlights) {
+        echo "<p>";
+        echo _("The following highlights have been used:") . "<br>";
+        foreach ($highlights as $hl_class => $message) {
+            echo "<span class='$hl_class'>$message</span><br>";
+        }
+        echo "</p>";
+    }
+
+    echo "</div>";
+    // here ends the div for the WordCheck header
+
+    echo '<table id="tbtext">';
+
+    // print the link to the WordCheck FAQ
+    echo '<tr><td id="tdtop">';
+    echo new_window_link(get_faq_url("wordcheck-faq.php"), _("WordCheck FAQ"));
+    echo '</td></tr>';
+
+    echo '<tr>';
+    echo '<td id="tdtext">';
+    $ppage->echo_hidden_fields();
+    echo $page_contents;
+    echo '</td>';
+    echo '</tr>';
+
+    echo '<tr>';
+    echo '<td id="tdtop">';
+    $value = attr_safe(_("Submit Corrections"));
+    $title = attr_safe($word_check_messages['keepCorrectons']);
+    echo "<input type='submit' name='spcorrect' id='spcorrect' value='$value' title='$title'>";
+    echo "&nbsp;&nbsp;&nbsp;";
+
+    $value = attr_safe(_("Quit WordCheck"));
+    $title = attr_safe(_("Abandon corrections and return to proofreading this page"));
+    echo "<input type='submit' name='spexit' value='$value' title='$title' onClick='return confirmExit();'>";
+
```

Comment:
There was formerly a space between these buttons caused by the white space in the raw html.